libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
I'd accept `reset_current_node` or `reset_this_node`. `rmq` or `rabbitmq` are absolutely unnecessary and `self` is less specific than `current_node`.